projects/OPT/download_opt175b.md

```diff
@@ -43,26 +43,29 @@ done
 ```
 Note that most of our models expect to run with Model (Tensor) Parallelism. For smaller models, some
 users may find it easier to eliminate model parallelism. The checkpoints can be converted
-to eliminate use of MP with the `consolidate_fsdp_shards.py` script:
+to eliminate use of MP with the `reshard_mp.py` script:
 
 ```bash
-python metaseq.scripts.consolidate_fsdp_shards ${FOLDER_PATH}/checkpoint_last --new-arch-name transformer_lm_gpt --save-prefix ${FOLDER_PATH}/consolidated
+python -m metaseq.scripts.reshard_mp \
```

Comment:
This doesn't seem to match the args from reshard_mp? https://github.com/facebookresearch/metaseq/blob/071b17aad97e7257cc026cf5d03492520bec5b91/metaseq/scripts/reshard_mp.py#L76

Am I missing something?

Reply:
Ah, that's probably an old commit. Please see [the one](https://github.com/facebookresearch/metaseq/blob/main/metaseq/scripts/reshard_mp.py#L17) in `main` for some updates after PR #556 got merged.